<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindows</class>
 <widget class="QMainWindow" name="MainWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow::separator:horizontal {
    width: 3px;
    margin-top: 3px;
    margin-bottom: 3px;
}

QMainWindow::separator:vertical {
    height: 3px;
    margin-left: 3px;
    margin-right: 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget_main">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Home">
         <attribute name="title">
          <string>Home</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="Auto_test">
         <attribute name="title">
          <string>Auto Test</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QFrame" name="Auto_test_frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QTreeView" name="Run_case_status_tree"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuNew_Project_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project_File"/>
    <addaction name="actionOpen_Project"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="action_SetPythonFileHome"/>
   </widget>
   <addaction name="menuNew_Project_File"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
	
	background-color: rgb(244, 255, 240);
    border-bottom: 1px solid #19232D;
    padding: 2px;
    font-weight: bold;
}

QToolBar QToolButton{
    background-color: rgb(244, 255, 240);
}</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Auto_Test"/>
   <addaction name="ScriptConfigure"/>
   <addaction name="TestPlan"/>
   <addaction name="separator"/>
   <addaction name="Test_Instrument"/>
   <addaction name="TestProduct"/>
   <addaction name="separator"/>
   <addaction name="action_Script_Editor_Pycharm"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="DW_ToolDock">
   <property name="styleSheet">
    <string notr="true">
QDockWidget::title {
    padding:6px 1px 1px;   /* better size for title bar */
    border: none;
    background-color: #f2fdee;
}
QDockWidget::dockWidgetContents {
	
	border: 1px solid black;

}
QDockWidget::separator {
    width: 3px;
}


</string>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Operation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QToolBox" name="toolBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolBox {
    padding: 0px;
    border: 1px solid 
}

QToolBox::selected {
    padding: 0px;
    border: 2px solid #1464A0;
}

QToolBox::tab {
    border: 1px solid 
    color: #F0F0F0;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

QToolBox::tab:disabled {
    color: #787878;
}

QToolBox::tab:selected {
    border-bottom: 2px solid
}

QToolBox::tab:!selected {
    border-bottom: 2px solid 
}

QToolBox::tab:selected:disabled {
    border-bottom: 2px solid #14506E;
}

QToolBox::tab:!selected:disabled {
    background-color: #19232D;
}

QToolBox::tab:hover {
    border-color: #148CD2;
    border-bottom: 2px solid #148CD2;
}

QToolBox QScrollArea QWidget QWidget {
    padding: 0px;
    background-color: #19232D;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>5</number>
       </property>
       <property name="midLineWidth">
        <number>3</number>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="TB_Routerpage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>311</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Router Tool</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="TB_ConfigurePage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>98</width>
          <height>311</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Configure Tool</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="DW_Proterty">
   <property name="styleSheet">
    <string notr="true">QDockWidget {
    outline: 1px solid #32414B;
    background-color: #19232D;
    border: 1px solid #32414B;
    border-radius: 4px;
}

QDockWidget::title {
    padding:6px 1px 1px;   /* better size for title bar */
    border: none;
    background-color: #32414B;
}

QDockWidget::close-button {
    background-color: #32414B;
    border-radius: 4px;
    border: none;
}

QDockWidget::close-button:hover {
    border: 1px solid #32414B;
}

QDockWidget::close-button:pressed {
    border: 1px solid #32414B;
}

QDockWidget::float-button {
    background-color: #32414B;
    border-radius: 4px;
    border: none;
}

QDockWidget::float-button:hover {
    border: 1px solid #32414B;
}

QDockWidget::float-button:pressed {
    border: 1px solid #32414B;
}</string>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Property</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="1" column="0">
        <widget class="QListView" name="LV_ProPerty">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Property</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Opeartioninfodisp">
   <property name="styleSheet">
    <string notr="true">QDockWidget {
    outline: 1px solid #32414B;
    background-color: #19232D;
    border: 1px solid #32414B;
    border-radius: 4px;
}

QDockWidget::title {
    padding:7px 1px 1px;   /* better size for title bar */
    border: none;
    background-color: #32414B;
}

QDockWidget::close-button {
    background-color: #32414B;
    border-radius: 4px;
    border: none;
}

QDockWidget::close-button:hover {
    border: 1px solid #32414B;
}

QDockWidget::close-button:pressed {
    border: 1px solid #32414B;
}

QDockWidget::float-button {
    background-color: #32414B;
    border-radius: 4px;
    border: none;
}

QDockWidget::float-button:hover {
    border: 1px solid #32414B;
}

QDockWidget::float-button:pressed {
    border: 1px solid #32414B;
}</string>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Operation  Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="0">
        <widget class="QTabWidget" name="TW_OpeationDisp">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tw_opeartioninfo">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(246, 255, 240);</string>
          </property>
          <attribute name="icon">
           <iconset resource="HomePage.qrc">
            <normaloff>:/StartForm/Resources/operation.png</normaloff>:/StartForm/Resources/operation.png</iconset>
          </attribute>
          <attribute name="title">
           <string>Opeartion Info</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_ErrorInfo">
   <property name="styleSheet">
    <string notr="true">QDockWidget {
    outline: 1px solid #32414B;
    background-color: #19232D;
    border: 1px solid #32414B;
    border-radius: 4px;
}

QDockWidget::title {
    padding:7px 1px 1px;   /* better size for title bar */
    border: none;
    background-color: #32414B;
}

QDockWidget::close-button {
    background-color: #32414B;
    border-radius: 4px;
    border: none;
}

QDockWidget::close-button:hover {
    border: 1px solid #32414B;
}

QDockWidget::close-button:pressed {
    border: 1px solid #32414B;
}

QDockWidget::float-button {
    background-color: #32414B;
    border-radius: 4px;
    border: none;
}

QDockWidget::float-button:hover {
    border: 1px solid #32414B;
}

QDockWidget::float-button:pressed {
    border: 1px solid #32414B;
}</string>
   </property>
   <property name="windowTitle">
    <string>Error Info</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTabWidget" name="TW_OpeationDisp_ErrorInfo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(65, 65, 65);</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tw_errorinfo_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 246, 222);</string>
        </property>
        <attribute name="icon">
         <iconset resource="HomePage.qrc">
          <normaloff>:/StartForm/Resources/Error.png</normaloff>:/StartForm/Resources/Error.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Error Info</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Terminal">
   <property name="styleSheet">
    <string notr="true">QDockWidget {
    outline: 1px solid #32414B;
    background-color: #19232D;
    border: 1px solid #32414B;
    border-radius: 4px;
}

QDockWidget::title {
    padding:7px 1px 1px;   /* better size for title bar */
	border:  none;
    background-color: #32414B;
}

QDockWidget::close-button {
    background-color: #32414B;
    border-radius: 4px;
    border: none;
}

QDockWidget::close-button:hover {
    border: 1px solid #32414B;
}

QDockWidget::close-button:pressed {
    border: 1px solid #32414B;
}

QDockWidget::float-button {
    background-color: #32414B;
    border-radius: 4px;
    border: none;
}

QDockWidget::float-button:hover {
    border: 1px solid #32414B;
}

QDockWidget::float-button:pressed {
    border: 1px solid #32414B;
}</string>
   </property>
   <property name="windowTitle">
    <string>Run Terminal</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <layout class="QGridLayout" name="gridLayout_14">
     <item row="0" column="0">
      <widget class="QTabWidget" name="TW_OpeationDisp_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tw_Terminal">
        <attribute name="icon">
         <iconset resource="HomePage.qrc">
          <normaloff>:/StartForm/Resources/terminal.png</normaloff>:/StartForm/Resources/terminal.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Terminal</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>617</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPlainTextEdit" name="PTE_TerimnalDisplayArea">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(65, 65, 65);
font: 75 12pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_clearTerminal">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="HomePage.qrc">
             <normaloff>:/StartForm/Resources/clean up.png</normaloff>:/StartForm/Resources/clean up.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="Auto_Test">
   <property name="icon">
    <iconset resource="HomePage.qrc">
     <normaloff>:/StartForm/Resources/auto_test.png</normaloff>:/StartForm/Resources/auto_test.png</iconset>
   </property>
   <property name="text">
    <string>Auto Test</string>
   </property>
   <property name="toolTip">
    <string>Auto Test</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="ScriptConfigure">
   <property name="icon">
    <iconset resource="HomePage.qrc">
     <normaloff>:/StartForm/Resources/config_script.png</normaloff>:/StartForm/Resources/config_script.png</iconset>
   </property>
   <property name="text">
    <string>ScriptConfigure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="TestPlan">
   <property name="icon">
    <iconset resource="HomePage.qrc">
     <normaloff>:/StartForm/Resources/test plan.png</normaloff>:/StartForm/Resources/test plan.png</iconset>
   </property>
   <property name="text">
    <string>TestPlan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="Test_Instrument">
   <property name="icon">
    <iconset resource="HomePage.qrc">
     <normaloff>:/StartForm/Resources/instrment.png</normaloff>:/StartForm/Resources/instrment.png</iconset>
   </property>
   <property name="text">
    <string>Test Instrument</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="TestProduct">
   <property name="icon">
    <iconset resource="HomePage.qrc">
     <normaloff>:/StartForm/Resources/rru.png</normaloff>:/StartForm/Resources/rru.png</iconset>
   </property>
   <property name="text">
    <string>Test Product</string>
   </property>
  </action>
  <action name="action_Script_Editor_Pycharm">
   <property name="icon">
    <iconset resource="HomePage.qrc">
     <normaloff>:/StartForm/Resources/ide-data.png</normaloff>:/StartForm/Resources/ide-data.png</iconset>
   </property>
   <property name="text">
    <string>Script Editor(Pycharm)</string>
   </property>
  </action>
  <action name="action_SetPythonFileHome">
   <property name="icon">
    <iconset resource="HomePage.qrc">
     <normaloff>:/StartForm/Resources/Path.png</normaloff>:/StartForm/Resources/Path.png</iconset>
   </property>
   <property name="text">
    <string>SetPythonFileHome</string>
   </property>
   <property name="toolTip">
    <string>select  the python  file path which you want to choose  </string>
   </property>
  </action>
  <action name="actionNew_Project_File">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HomePage.qrc"/>
 </resources>
 <connections/>
</ui>
